'npm run build-prod' fails on Windows

### Expected behaviour
'npm run build-prod'  should work on Windows
### Actual behaviour
'npm run build-prod' on Windows fails with
```
Vit@DESKTOP-VMV5UOP MINGW64 ~/git/lisk-hub (development)
$ npm run build-prod

> lisk-hub@0.1.0 build-prod C:\Users\Vit\git\lisk-hub
> webpack --config ./config/webpack.config.prod --env.prod

C:\Users\Vit\git\lisk-hub\config\webpack.config.react.js:15
  langs.forEach((item) => {
       ^

TypeError: Cannot read property 'forEach' of null
    at getLocales (C:\Users\Vit\git\lisk-hub\config\webpack.config.react.js:15:8)
    at Object.<anonymous> (C:\Users\Vit\git\lisk-hub\config\webpack.config.react.js:22:19)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\Vit\git\lisk-hub\config\webpack.config.prod.js:7:21)

npm ERR! Windows_NT 10.0.15063
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build-prod"
npm ERR! node v6.11.3
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! lisk-hub@0.1.0 build-prod: `webpack --config ./config/webpack.config.prod --env.prod`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the lisk-hub@0.1.0 build-prod script 'webpack --config ./config/webpack.config.prod --env.prod'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the lisk-hub package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     webpack --config ./config/webpack.config.prod --env.prod
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs lisk-hub
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls lisk-hub
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\Vit\git\lisk-hub\npm-debug.log
```

### Steps to reproduce
run 'npm run build-prod' on Windows git bash

--- a/config/webpack.config.react.js
+++ b/config/webpack.config.react.js
@@ -11,7 +11,7 @@ const path = require('path');
 const getLocales = (url) => {
   const file = fs.readFileSync(path.join(__dirname, url));
   const str = [];
-  const langs = file.toString().match(/.*:\s{\n/g);
+  const langs = file.toString().match(/.*:\s{\r?\n/g);
   langs.forEach((item) => {
     str.push(item.match(/[a-z]{2}/g)[0]);
   });

